apps/frontend/src/modules/files/slices/actions.ts

```diff
@@ -0,0 +1,23 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+import { type AsyncThunkConfig } from "~/libs/types/types.js";
+
+import { NotificationMessage } from "../libs/enums/enums.js";
+import { type UserFileResponseDto } from "../libs/types/type.js";
+import { name as sliceName } from "./files.slice.js";
+
+const updateUserAvatar = createAsyncThunk<
+	UserFileResponseDto,
```

Comment:
```suggestion
	UserFileResponseDto,
```
here we should use an auth-user type (and add avatar to this type)